package.json

```diff
@@ -1,10 +1,17 @@
 {
-    "name": "vertigis-workflow-project",
-    "private": true,
-    "version": "1.0.0",
-    "description": "",
-    "license": "UNLICENSED",
-    "author": "",
+    "name": "@vertigis/workflow-activities-inline",
+    "version": "0.0.0-semantically-released",
+    "description": "VertiGIS Studio Workflow activities for VertiGIS Inline",
+    "license": "MIT",
```

Comment:
What are the implication of changing from unlicensed to MIT?

Reply:
The MIT license is what we use for the other projects. As far as I am aware it means others can use and modify this project